lib/dma.c

```diff
@@ -256,6 +256,33 @@ dma_map_region(dma_controller_t *dma, dma_memory_region_t *region)
     return 0;
 }
 
+static ssize_t
+get_bitmap_size(size_t region_size, size_t pgsize)
+{
+    if (pgsize == 0) {
```

Comment:
nit, can these both be asserts, then you can return size_t ?

Reply:
We can, but `get_bitmap_size` has two callers so we'd have to duplicate checks there and I'd like to avoid that. However, in case of error we can return 0 therefore we _can_ change the return type to `size_t`. How does this sound?

Reply:
I think it's fine as is, then.